src/main/resources/application.yml

```diff
@@ -23,6 +23,18 @@ spring:
       resourceserver:
         jwt:
           jwk-set-uri: ${oauth.endpoint.url}/.well-known/jwks.json
+      client:
+        registration:
+          basm-api:
+            provider: basm-auth
+            client-id: ${basm.client.client-id}
+            client-secret: ${basm.client.client-secret}
+            authorization-grant-type: client_credentials
```

Comment:
basm is its own authorization server and has its own set of credentials - rather than using auth